tests/istioUtils.sh

```diff
@@ -25,9 +25,14 @@ function create_rule() {
       || error_exit 'Could not create rule'
 }
 
+function replace_rule() {
+    ${ISTIOCLI} -n ${NAMESPACE} replace -f ${1} \
+      || error_exit 'Could not create rule'
+}
+
 function cleanup_all_rules() {
     print_block_echo "Cleaning up rules"
-    local rules=($(${ISTIOCLI} -n ${NAMESPACE} list route-rule \
+    local rules=($(${ISTIOCLI} -n ${NAMESPACE} get route-rule \
```

Comment:
If you use this to test new builds (which I think this is the main purpose), you should/must also use the `-i` argument to specify a compatible istioctl. So, I'd say it's ready to merge.